### DISCUSSION TITLE
Welcome to OpenSLO Discussions!

### DISCUSSION THREAD

**Discussion by programmer04** [OpenSLO/OpenSLO#251] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by bilyboy785** [comment]
Hi ! Where i can find a list of all compatible datasources ?
